cloudlib/logger.py

```diff
@@ -171,7 +172,7 @@ def return_logfile(filename, log_dir='/var/log'):
         :param log_dir: ``str``
         :return: ``str``
         """
-        user = os.getuid()
+        user = getpass.getuser()
```

Comment:
The explicite change modifies the return data such that on a *nix platform the method would use the "username" instead of the "userID" In this case can we do a conditional load? 

add ``sys`` to the above imports and then

``` python
if sys.platform == 'win32':
    user = getpass.getuser()
else:
    user = os.getuid()
```